packages/amplify-cli-core/src/feature-flags/featureFlags.ts

```diff
@@ -515,6 +515,12 @@ export class FeatureFlags {
         defaultValueForExistingProjects: false,
         defaultValueForNewProjects: true,
       },
+      {
+        name: 'improvePluralization',
+        type: 'boolean',
+        defaultValueForExistingProjects: false,
+        defaultValueForNewProjects: true,
```

Comment:
If we set the flag to true for new projects, they immediately won't work with the existing iOS library. This is a breaking change, right? 
before: 
- `type Wish` generates `listWishs` 
- `type Wishes` generates `listWishess`

after:
- `type Wish` generates `listWishes` 
- `type Wishes` generates `listWishes`

How do new projects work with existing iOS library? Is it because iOS library has logic coupled to how the name is generated, and shoudn't have? Should we have feature flags for the libraries as well?